HeterogeneousCore/CUDAUtilities/src/requireCUDADevices.cc

```diff
@@ -0,0 +1,28 @@
+#include <cstdlib>
+#include <iostream>
+
+#include <cuda_runtime.h>
+
+#include "HeterogeneousCore/CUDAUtilities/interface/requireCUDADevices.h"
+
+bool hasCUDADevices() {
+  int devices = 0;
+  auto status = cudaGetDeviceCount(&devices);
+  if (status != cudaSuccess) {
+    std::cerr << "Failed to initialise the CUDA runtime, the test will be skipped."
```

Comment:
Is this called from within cmsRun? If so, the use of `std::cerr` is not standard.

Reply:
No, it is intended for unit tests only. I'll add a comment to the header.